cmd/openshift-install/create.go

```diff
@@ -288,6 +291,9 @@ func newCreateCmd(ctx context.Context) *cobra.Command {
 	for _, t := range targets {
 		t.command.Args = cobra.ExactArgs(0)
 		t.command.Run = runTargetCmd(ctx, t.assets...)
+		if "Cluster" == t.name {
+			cmd.PersistentFlags().BoolVar(&skipPasswordPrintFlag, "skip-password-print", false, "Do not print the generated user password.")
```

Comment:
```suggestion
			cmd.PersistentFlags().BoolVar(&skipPasswordPrintFlag, "skip-password-print", true, "Do not print the generated user password.")
```

Don't we want to default this to true to skip the password printing by default?

Reply:
@barbacbd yoda condition fixed, thanks.
But I'd say it's safer to leave false as the default, at least for one release, as user scripts/hacks may depend on that.

Reply:
I am fine with that, thanks.

Reply:
> But I'd say it's safer to leave false as the default, at least for one release, as user scripts/hacks may depend on that.
Fixing user scripts sooner rather than later is a good idea. Also 4.16 is a LTS release. So, let us swap the defaults now.

Reply:
4.16 where this will land is a LTS release. So, it is better to have the default to true to skip printing the password sooner rather than a later release.
Also, can we call the the flag `printUserPassword` which is set to false by default? User scripts can be updated to specifically turn on password printing.